pkg/operator/controllers/monitoring/monitoring_controller.go

```diff
@@ -99,22 +96,27 @@ func (r *Reconciler) Reconcile(ctx context.Context, request ctrl.Request) (ctrl.
 	return reconcile.Result{}, nil
 }
 
-func (r *Reconciler) reconcilePVC(ctx context.Context, request ctrl.Request) (ctrl.Result, error) {
-	pvcList, err := r.kubernetescli.CoreV1().PersistentVolumeClaims(monitoringName.Namespace).List(ctx, metav1.ListOptions{LabelSelector: prometheusLabels})
+func (r *Reconciler) reconcilePVC(ctx context.Context) (ctrl.Result, error) {
+	pvcList := &corev1.PersistentVolumeClaimList{}
+	selector, _ := labels.Parse(prometheusLabels)
```

Comment:
I'm ok with ignoring this error since the labels are hardcoded and the current formatting should not result in an error.